src/styles/index.css

```diff
@@ -0,0 +1,13 @@
+@import "./block/card.css";
+@import "./block/stars.css";
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset margins, padings with `*`, list all tags to you need reset
```
ul,
li,
p ... {
  m: 0;
  p: 0;
}
```


```suggestion
```

Reply:
understood thanks